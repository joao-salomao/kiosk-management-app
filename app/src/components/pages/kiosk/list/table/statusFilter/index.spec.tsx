import renderer from 'react-test-renderer';
import { RecoilRoot } from 'recoil';
import { StatusFilter } from ".";


it('renders correctly', () => {
  const tree = renderer
    .create(
      <RecoilRoot>
        <StatusFilter />
      </RecoilRoot>
    )
    .toJSON();

  expect(tree).toMatchSnapshot();
});