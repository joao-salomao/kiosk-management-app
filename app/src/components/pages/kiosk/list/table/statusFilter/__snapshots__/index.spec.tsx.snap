// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <label
    htmlFor="statusFilter"
  >
    Status filter
  </label>
  <br />
  <select
    className="block p-2 w-full text-sm text-gray-900 bg-gray-50 rounded-lg border border-gray-300 focus:ring-blue-500 focus:border-blue-500"
    name="statusFilter"
    onChange={[Function]}
    value="all"
  >
    <option
      value="all"
    >
      All Kiosks
    </option>
    <option
      value="open"
    >
      Opened Kiosks
    </option>
    <option
      value="closed"
    >
      Closed Kiosks
    </option>
  </select>
</div>
`;
